lib/src/text-input/TextInput.stories.tsx

```diff
@@ -0,0 +1,418 @@
+import React from "react";
+import { userEvent, within, waitFor } from "@storybook/testing-library";
+import { fireEvent } from "@testing-library/react";
+import { BackgroundColorProvider } from "../BackgroundColorContext";
+import Title from "../../.storybook/components/Title";
+import ExampleContainer from "../../.storybook/components/ExampleContainer";
+import DarkContainer from "../../.storybook/components/DarkSection";
+import DxcTextInput from "./TextInput";
+
+export default {
+  title: "Text input",
+  component: DxcTextInput,
+};
+
+const action = {
+  onClick: () => {
+    console.log("Copy that!");
```

Comment:
I think we can remove this 'console.log' and leave an empty function.

Reply:
Agree! Done!